tools/ovc/unit_tests/ovc/utils/cli_parser_test.py

```diff
@@ -343,12 +343,81 @@ def test_mo_convert_params(self):
         assert argv.input_model == args['input_model']
         assert argv.extension == args['extension']
         assert argv.input == ['name', ("a", [1,2,3], ov.Type.f32)]
-        assert argv.output == ["a","b","c"]
+        assert argv.output == ["a", "b", "c"]
 
         for arg, value in vars(argv).items():
             if arg not in args and arg != 'is_python_api_used':
                 assert value == cli_parser.get_default(arg)
 
+    def test_output_post_parsing_1(self):
+        args = {'input_model': os.path.dirname(__file__),
+                'input': "input_1[1,2,3]",
+                'output_model': os.getcwd() + "model.xml",
+                "framework": "onnx",
```

Comment:
Why do we need to write this parameter? `ovc` doesn't have it as a publicly visible.

Reply:
This is a very good question! Unfortunately without this attribute `arguments_post_parsing` was failing because inside it calls `params_parsing` which checks `argv.framework`. Even in case of using `convert_model` is already initialized for us.

I didn't like introducing such a redundant attribute into tests. I detaste to such dangling placeholding attributes introduced for only reason that the test does not fall, i made check for `arvg.framework` safer and removed it:
https://github.com/openvinotoolkit/openvino/pull/19391/files#diff-fea8ab7df43344bcc52da931836648c9e47bf0fd173be1961d35d2718cbcb11aR364-R365

PS: Unfortunately even `convert_model` is bound to cli parser because `convert_model` uses `_convert(..)` which has a lot of references to cli_parser inside itself. Probably it would be better move all parser things into cli tool `__main__` https://github.com/openvinotoolkit/openvino/blob/f0300a36eb42115181a1509c1ca82324e81556cf/tools/ovc/openvino/tools/ovc/main.py#L19-L32 but for the moment we don't have time for that, imho.